docs/recipes/browser-testing.md

```diff
@@ -0,0 +1,69 @@
+# Setting up AVA for browser testing
+
+AVA does not support running tests in browsers yet.
+Some libraries require ```window``` object is defined.
```

Comment:
This line could be better:

> Some libraries require browser specific globals (`window`, `document`, `navigator`, etc).

Also, You just need a single tick around `window` (not triple).